GitUI/Configuring/Shell.cs

```diff
@@ -0,0 +1,32 @@
+﻿namespace GitUI.Configuring
+{
+    public interface IShell
```

Comment:
Why do we need this?
There's already `IShellDescriptor` and each shell has its own definition file.

Reply:
Like in math. To simplify it is not always necessary to remove, sometimes it is necessary to add. 😄 
Refactoring later